x-pack/plugins/alerts/server/saved_objects/index.ts

```diff
@@ -42,6 +42,19 @@ export function setupSavedObjects(
     mappings: mappings.alert,
   });
 
+  savedObjects.registerType({
+    name: 'invalidate_pending_api_key',
+    hidden: false,
```

Comment:
I wonder if this should be hidden instead? Kibana users will have CRUD access to these objects using the SO APIs. I guess feature privileges will protect here but it won't when security is disabled.

```suggestion
    hidden: true,
```